tools/dbtool.py

```diff
@@ -190,6 +190,7 @@ def load_into_dict(filename, settings):
     "char_equip.sql",
     "char_equip_saved.sql",
     "char_exp.sql",
+    "char_fishing_contest_history",
```

Comment:
Is this likely to get very large over time, and need truncating?

Reply:
it shouldn't -- it's one row per character at most.  This is like `char_history`, but only holds the number of times each character has come in either first, second, third, or fourth-tenth place.  It is _not_ an ongoing log of individual contest results.  It will look something like this:

![image](https://github.com/user-attachments/assets/e0fbd46d-d8c9-4736-b282-c548a7e75586)